docs/examples/embed.js

```diff
@@ -0,0 +1,65 @@
+/*
+	An example of how you can send embeds
+*/
+
+// Extract the required objects from the discord.js module
+const { Client, MessageEmbed } = require('discord.js');
+
+// Create an instance of a Discord client
+const client = new Client();
+
+// The token of your bot - https://discordapp.com/developers/applications/me
+const token = 'your bot token here';
+
+// The ready event is vital, it means that your bot will only start reacting to information
+// from Discord _after_ ready is emitted
+client.on('ready', () => {
+  console.log('I am ready!');
+});
+
+client.on('message', message => {
+  // If the message is "how to embed"
+  if (message.content === 'how to embed') {
+    // We can create embeds using the MessageEmbed constructor
+    // Read more about all that you can do with the constructor in the docs
+    const embed = new MessageEmbed()
+      .setTitle(`A slick little embed`) // Set the author of the field
+      .setColor('RANDOM') // Set the color of the embed
+      .setDescription(`Hello, this is a slick embed!`); // Set the main content of the embed
+    // Send the embed to the same channel as the message
+    message.channel.send({ embed });
```

Comment:
isn't really needed anymore since 11.2.1 you can just do `message.channel.send(embed)` and that prevents confusion for newbies since it isn't required to be named "embed" then and could be named anything